integration/spark/src/main/scala/org/apache/carbondata/indexserver/DistributedPruneRDD.scala

```diff
@@ -145,6 +145,7 @@ private[indexserver] class DistributedPruneRDD(@transient private val ss: SparkS
         while (reader.nextKeyValue()) {
           blocklets.add(reader.getCurrentValue)
         }
+        reader.close()
```

Comment:
no need to call close. nextKeyValue is already handling

Reply:
removed